scripts/vim-patch.sh

```diff
@@ -87,7 +87,7 @@ commit_message() {
 }
 
 find_git_remote() {
-  git remote -v | awk '/neovim\/neovim.*(fetch)/{print $1}'
+  git remote -v | awk '$2 ~ /neovim\/neovim/ && $3 == "(fetch)" {print $1}'
```

Comment:
Depending on how strict you want to be, it may be worth changing the first match to `$2 ~ /github.com[:/]neovim\/neovim/`.